### PR TITLE
FortiManager Plugin Module Conversion: fmgr_ha 

### PR DESCRIPTION
**Pull request by ftntcorecse**
##### SUMMARY

Contains code changes for fmgr_device to use the new plugin connection. Code UPDATE. NOT a new module.

##### ISSUE TYPE

- Bugfix Pull Request

##### COMPONENT NAME

fmgr_ha.py
test_fmgr_ha.py
test_fmgr_ha.json
##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: lib/ansible/modules/network/fortimanager/fmgr_ha.py [hunk 1]
```diff
@@ -288,37 +250,40 @@ def main():
...
         # IF THE PEER SN DEFINED, BUT THE IPS ARE NOT, THEN QUIT
         if paramgram["fmgr_ha_peer_sn"] is not None:
             # CHANGE GET_HA_PEERS TO SHOW INTENT TO EDIT PEERS
             get_ha_peers = 1
             # DOUBLE CHECK THAT THE REST OF THE NEEDED PARAMETERS ARE THERE
             if paramgram["fmgr_ha_peer_ipv4"] is None and paramgram["fmgr_ha_peer_ipv6"] is None:
-                fmgr_logout(fmg, module, msg="HA Peer Serial Number is defined but the "
-                                             "IPv4 and IPv6 fields are empty."
-                                             " Fill in the IPv4 or v6 parameters in the playbook")
+                module.exit_json(msg="HA Peer Serial Number is defined but the "
```
Comment (NilashishC) [inline lib/ansible/modules/network/fortimanager/fmgr_ha.py]: Can `required_together` be used to handle these cases instead of `exit_json` calls?

Comment (ftntcorecse) [reply lib/ansible/modules/network/fortimanager/fmgr_ha.py]: Ended up using required_if, but yes we were able to remove about 35 lines worth of validation code in favor of required_if.

Comment (NilashishC) [reply lib/ansible/modules/network/fortimanager/fmgr_ha.py]: Great!

**Review by NilashishC — APPROVED** [review]
LGTM

**Pull request merged**